Overal visual changes

rmaalmeida opened the pull request:
Visual changes to version 1.3
- ligthter colors,
- run/stop buttons sunken when pressed
- changed order of buttons on the bottom

rmaalmeida committed:
Overal visual changes

Visual changes to version 1.3
- ligthter colors,
- run/stop buttons sunken when pressed
- changed order of buttons on the bottom

rmaalmeida committed:
Delete alice-last-config.cfg

rmaalmeida committed:
Delete alice_init.ini

rmaalmeida committed:
Visual Changes and Variables Refactor

Refator most of the main screen widgets variable and change the visuals for a flat option

rmaalmeida committed:
Merge branch 'Version-1.3' of https://github.com/rmaalmeida/alice into Version-1.3

damercer commented:
Thanks for suggesting these changes.

Since the default colors can (should be changed) by users in the alice_init.ini your suggested color changes will not be included as the defaults in master source code. User by user preferences can not be accommodated in the official release packages.

The other visual changes will be considered.

Changes to widget locations in the UI screens are problematic in that it will require all the graphics in the User Guides be changed and possibly confuse existing users and force users to likely require changes to Lab documents.

rmaalmeida committed:
Revert "Merge branch 'Version-1.3' of https://github.com/rmaalmeida/alice into Version-1.3"

This reverts commit 08985de14e078ba83827d967c499c105d91c3661, reversing
changes made to 08972400b01dcc944d7382690296ee9d76c1b1f2.

rmaalmeida committed:
Revert "Visual Changes and Variables Refactor"

This reverts commit 08972400b01dcc944d7382690296ee9d76c1b1f2.

rmaalmeida committed:
Simple Interface Version

Implemented a simple interface version with only scope, XY and analog wave generator. All advanced controls are disabled. AWG and Scope are on the same screen.

damercer commented:
Hi:
Please remember you are submitting pull request that fundamentally change the master copy of the source code through this means. If you just want to show the developers some personal changes you have made, then post a plain issue listing the changes / enhancements that you are suggesting, and include a link to your personal copy of the source code.

I can't merge pull requests like these (with the same name as the "golden" source file) in to the master copy that will be distributed to the wider user base.

As to your latest specific ideas in this request. You just need to be patient and wait for the next official release which will implement an option to combine scope and AWG controls into the main window with the XY plot in a separate window. The first pass at a method to localize the language through an optional configuration file will also be included.

Please limit pull requests to fatal bug fixes.
